src/pre_defined_classes/mathematical_program/medium/q4/BasketballGames.java

```diff
@@ -0,0 +1,28 @@
+package pre_defined_classes.mathematical_program.medium.q4;
+
+/*
+You are a scorekeeper for your high school basketball team. It’s the end of the season and you have to calculate your
+team’s participation in all of the tournaments. You decide to do this using a Java program. Your job is to get the
+ program to calculate the total score across all the games, find the highest score, and find the lowest score.
+ Here are the game statistics:
+            Game #1 - Your school scored 36 points
+            Game #2 - Your school scored 59 points
+            Game #3 - Your school scored 32 points
+	Sample Output:
+		Our team scored 100 points in the season!
```

Comment:
Not an issue with your code really, but I could see how the conflicting information on the number of scored points could lead to some confusion.

Reply:
Like my sample output?

Reply:
Yeah because I got a bit confused reading it, when the minimum was 20 points when 20 wasn't given in the question

Reply:
I just was not sure I wanted to hand them the total complete output, that might make it too easy

Reply:
true, maybe replace the numbers with like, xxx or something

Reply:
Is that not more confusing?

Reply:
maybe? The question needs clarification in my opinion but I'm not sure how to go about it

Reply:
I changed the question up again if you want to take a look again